src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/ReadPosRankSumTestUnitTest.java

```diff
@@ -92,7 +92,7 @@ public void testReadPos(){
         Assert.assertTrue(annotatePastEnd.isEmpty());
     }
 
-    @Test
+    @Test(enabled = false)  //To be consistent with annotate raw data now produces the the ranksum, making this test incorrect
```

Comment:
?

Reply:
I moved the test to the test suite for the allele specfic version and I will fix it

Reply:
The problem was that GATK3 changed the behavior of the annotation so annotateRawData produces the finalized output now

Reply:
@ldgauthier  Is this the behavior we want, now all the rank sum annotations calculate the mannWhitney score over their values, resulting in some strange results for the readpos rank sum results for example